reporters/graphql/umaSubgraph.js

```diff
@@ -35,13 +39,17 @@ function LIQUIDATION_EVENTS_FOR_EMP(empAddress) {
             }
           }
         }
+        totalCollateralDeposited
```

Comment:
wow this is great! such a simple query returns so much data. great work @nicholaspai

Reply:
downside is we have no idea how its calculated but it seems correct!